src/Form.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react'
+import Card from 'react-bootstrap/Card';
+
+export default class Form1 extends Component {
+
+    render() {
+        return (
+            <div>
+                <Card style={{ width: '32rem' }}>
+                
+                </Card>
+            </div>
+        )
+    }
```

Comment:
Good work getting started on Form here 👍  See if you can move some Form logic into this component.